.devcontainer/docker-compose.yml

```diff
@@ -23,7 +23,11 @@ services:
     environment:
       PGADMIN_DEFAULT_EMAIL: ${PGADMIN_DEFAULT_EMAIL:-pgadmin4@pgadmin.org}
       PGADMIN_DEFAULT_PASSWORD: ${PGADMIN_DEFAULT_PASSWORD:-admin}
+      PGADMIN_LISTEN_PORT: 5050
+    volumes:
+      - ./servers.json:/pgadmin4/servers.json
+      - ./pgadmin_files:/var/lib/pgadmin
```

Comment:
Good idea! I had the same 😋

Reply:
The sad part is, that "autologin" still doesn't work.
In the meantime I've found out that it's probably because .pgpass has the wrong permissions (it gets ignored if it isn't chowned to the user and chmoded to 600 but I have no Idea how to do that from docker compose.

Reply:
I can share you what I had planned to follow for implementing this:
https://technology.amis.nl/2020/01/02/pgadmin-in-docker-provision-connections-and-passwords/
In docker-compose world, maybe adding
- `user: "pgadmin:pgadmin"` and
-  `- ./pgadmin_files:/var/lib/pgadmin:ro`

could be enough. If not, we may have to create a Dockerfile to run the `chown` and `chmod` commands manually...

Reply:
> If not, we may have to create a Dockerfile to run the chown and chmod commands manually...

After fighting it a little more, I think this is the way to go.

Reply:
It actually wasn't.
The pgadmin setup script messes everything up if you prepare it in advance.
I solved it with a few lines of python now.